src/Tizen.NUI.Components/Controls/ScrollableBase.cs

```diff
@@ -329,25 +329,28 @@ public ScrollbarBase Scrollbar
             }
             set
             {
-                if (scrollBar)
+                if (value != null)
```

Comment:
If this code is added, the ScrollBar can not set to null. Is this the correct behavior?

Reply:
Updated. Please check again.
Thanks